src/v2/Apps/Auctions/Components/MyBids/MyBidsBidHeader.tsx

```diff
@@ -38,6 +38,7 @@ export const MyBidsBidHeader: React.FC<MyBidsBidHeaderProps> = ({ sale }) => {
             style={{ objectFit: "cover" }}
             src={sale.coverImage.resized.src}
             srcSet={sale.coverImage.resized.srcSet}
+            lazyLoad
```

Comment:
@The-Beez-Kneez - this simple prop on our `Image` component is how you make an image fade in when the image scrolls into view. Its super helpful for performance.